examples/defaulter-gen/Makefile

```diff
@@ -0,0 +1,15 @@
+TOOL=defaulter-gen
+
+test:
+	@if ! git diff --quiet HEAD; then \
+	    echo "FAIL: git client is not clean"; \
+	    false; \
+	fi
+	@go build -o /tmp/$(TOOL)
+	@PKGS=$$(go list ./output_tests/...  | paste -sd' '); \
+	/tmp/$(TOOL) --logtostderr --v=6 -i $$(echo $$PKGS | sed 's/ /,/g') -O zz_generated
```

Comment:
I copy-pasted it from https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/authorization/v1/types.go#L27. I can update both if you think it's necessary.